app/src/js/components/Executions/executions-list.js

```diff
@@ -28,9 +29,11 @@ const ExecutionsList = ({
   match,
   executions
 }) => {
-  const { list } = executions || { list: { meta: { count: 0 }, results: [] } };
   const { params } = match || {};
   const { collectionId, granuleId } = params;
+  const { map } = executions || {};
+  const list = map[granuleId] || initialState.list;
```

Comment:
Does the `map[granuleId]` have the same structure as initialState.list?

Reply:
I'm confused the `list` with `executions.list`.  Can we rename it to something like `granuleExecutionsList`?